bootstrap/092_service_account_setup.sql

```diff
@@ -40,6 +40,10 @@ returns varchar
 language sql
 as
 begin
+    -- Always call setup_external_functions to ensure the external_functions are re-created with the latest opscenter
+    -- token as Sundeck likely invalidated the previous token.
+    CALL admin.setup_external_functions('opscenter_api_integration');
```

Comment:
why call this here? I don't understand what is special about this proc? Maybe the name is just thorwing me off